public/js/bmlt_tabbed_map-public.js

```diff
@@ -251,6 +251,9 @@ const bmltTabbedMapJS = function ($) {
         listContent += "<br><i>Formats: </i>" + val.formats + "<br>";
       }
       if ((val.virtual_meeting_link) || (val.phone_meeting_number)) {
+        if (val.virtual_meeting_additional_info) {
```

Comment:
I didnt add this to previous if statement as i figured we wouldnt want to add if either of those two didnt exist but im fine with changing that assumption